src/app/services/endpoints.service.ts

```diff
@@ -26,4 +27,12 @@ export class EndpointsService {
   getEmployees(): Observable<Employee[]> {
     return this.http.get<Employee[]>(environment.apiUrl + "employee")
   }
+
+  getCategories(): Observable<CategoriesResponse> {
+    return this.http.get<CategoriesResponse>(environment.apiUrl + "Category") 
+  }
+  
+  deleteCategory(categoryId: number): Observable<unknown> {
+    return this.http.delete(environment.apiUrl + "Category" + "?categoryId=" + categoryId.toString());
+  }
```

Comment:
You can also try using "params" to send queryParams :)
`return this.http.delete(environment.apiUrl + `Category`, {
            params: {
                categoryId: categoryId.toString()
            }
        });`